cetmix_tower_server/wizards/cx_tower_command_execute_wizard_view.xml

```diff
@@ -101,7 +101,7 @@
                         help="Run code as it appears in 'Rendered code' in wizard and return to wizard. Result will not be logged"
                         class="oe_highlight"
                         groups="cetmix_tower_server.group_manager"
-                        attrs="{'invisible': ['|', ('result', '!=', False),('code', '=', False), ('show_servers', '=', True)]}"
+                        attrs="{'invisible': [('show_servers', '=', True),'&amp;', '|', ('result', '!=', False),('code', '=', False)]}"
```

Comment:
Domain syntax is incorrect:
`condition_1 AND  (condition_2 AND OR condition_3)`